src/arena/simulation.rs

```diff
@@ -441,15 +441,12 @@ impl HoldemSimulation {
     }
 
     fn needs_action(&self) -> bool {
-        // active but not this player
-        let mut other_active = self.game_state.player_active;
-        other_active.disable(self.game_state.to_act_idx());
+        let active_players = self.game_state.player_active;
```

Comment:
I think this is the correct logic, but can you add some tests that use the VecHistorian and or the StatsTrackingHistorian to show that this is correct.

Reply:
Added